File: src/main/resources/static/js/Entity.js
class Entity{
   rows =[];

    constructor(tableName) {
        this.tableName = tableName;
    }
   addRow(entityRow){
       let row = {
           key: entityRow.key,
           name: entityRow.name,
           type: entityRow.type
       }
       this.rows.push(row)
   }
}

File: src/main/java/tu/kielce/modelowaniebazydanych/controller/Entity.java
package tu.kielce.modelowaniebazydanych.controller;

import java.util.List;

public class Entity {
    private String tableName;
    private List<Row> rows;

    public Entity() {
    }

    public Entity(String tableName, List<Row> rows) {
        this.tableName = tableName;
        this.rows = rows;
    }

    public String getTableName() {
        return tableName;
    }

    public void setTableName(String tableName) {
        this.tableName = tableName;
    }

    public List<Row> getRows() {
        return rows;
    }

    public void setRows(List<Row> rows) {
        this.rows = rows;
    }
}
